### ISSUE TITLE
Adding sign in to Markdown Wiki

### ISSUE THREAD

**Issue by Ahmaddahshan** [open-learning-exchange/open-learning-exchange.github.io#146]
Adding sign in option which will direct you to this link http://127.0.0.1:5985/apps/_design/bell/MyApp/index.html

**Comment by dogi** [comment]
@Ahmaddahshan please describe in more detail where you wanna add this link

**Comment by Ahmaddahshan** [comment]
@dogi First step, under Nation Bell!!

**Comment by dogi** [comment]
If I did not kinda know what you wanted to describe, I would be right now  with your last comment even more confused then before

@Ahmaddahshan maybe you add a screenshot where you paint in the place for the link which you wanna add

**Comment by Ahmaddahshan** [comment]
<img width="476" alt="opl" src="https://cloud.githubusercontent.com/assets/19496672/16015131/3234f19a-3149-11e6-9977-26a5b11e238e.PNG">

**Comment by Ahmaddahshan** [comment]
It should go right under Nation Bell. i hope this look more clear now @dogi

**Comment by dogi** [comment]
yes it is ;)
that is a go

**Comment by sthak004** [comment]
Has this issue been dealt with? If so, I don't see it on my end.

**Comment by DrewPerlman** [comment]
@sthak004 This isn't as much an issue as it is a suggestion.

**Comment by sthak004** [comment]
@DrewPerlman Ah yes, you're right. I mixed up what I was trying to say.
@DrewPerlman @dogi I'm assuming that this is still something both of you would like to see done. I went ahead and looked at index.html however, I couldn't find a ```<nav>``` tag in ```<body>``` anywhere to see if I could add a *Sign In* link to the dropdown menu.

I was able to see the dropdown code when I did inspect element on the [homepage](http://open-learning-exchange.github.io/#!index.md).
